FileManipulator.Test/GlobalJsonManipulatorTest.cs

```diff
@@ -128,14 +128,14 @@ public async Task ThrowsWhenMsbuildSdksKeyHasIncorrectType()
             };
 
         public static TheoryData<string, GlobalJsonModification> GetModificationsToNewFile()
-            => new TheoryData<string, GlobalJsonModification>
+            => new ()
             {
                 {
                     $"{{{NewLine}" +
                     $"    \"msbuild-sdks\": {{{NewLine}" +
                     $"        \"{MsBuildSdkNugetPackageId}\": \"{MsBuildSdkVersion}\"{NewLine}" +
                     $"    }}{NewLine}" +
-                    $"}}",
```

Comment:
I think these `$`s are here for alignment purposes with the previous lines

Reply:
That was resharper :)

Reply:
Adressed in 850dd740556684d43fef0fe1b4d1a2b2b3402f2f